### PR TITLE
move content string -> JSON

### PR DESCRIPTION
**Pull request by sdomagala**

### CODE REVIEW

File: src/integration/preparePushResult.js [hunk 1]
```diff
@@ -135,24 +135,27 @@ function backup(from, to, tempLocation, notClonedRepositoriesFile, backupOperati
...
-    });
+    const arrOfTasks = notClonedRepositores
+      .filter((repo) => repo && repo.path)
+      .map( (repo) => _backupSection(from, to, tempLocation, repo.path) );
 
     async.series(arrOfTasks, cb);
   });
 }
 
+function _backupSection(from, to, tempLocation, item) {
```
Comment (przezor) [inline src/integration/preparePushResult.js]: Description pls

**Comment by przezor** [comment]
get rid of yarn lock file

File: src/integration/preparePushResult.js [hunk 2]
```diff
@@ -163,12 +175,17 @@ function eraseRepositoriesFromDest(tempLocation, indepenedentDocuRepositoriesFil
   const repoPath = `./${tempLocation}/${indepenedentDocuRepositoriesFile}`;
 
   reader.readFile(repoPath, (err, repoMatrix) => {
-    if (err || repoMatrix.length === 0) return cb();
+    const repoRawArray= JSON.parse(repoMatrix);
+    if (err || repoRawArray.length === 0) return cb();
```
Comment (mjasinski5) [inline src/integration/preparePushResult.js]: +    if (err || !repoRawArray.length) return cb();

File: src/integration/cloneDocuSources.js [hunk 3]
```diff
@@ -21,8 +21,10 @@ const eachRegTopic = require('../helpers/registryIterator'),
 function cloneDocuSources(registry, config, topics, next) {
 
   iterateRegClone(registry, config, topics, () => {
-    creator.createFile(`${config.tempLocation}/${config.indepenedentDocuRepositoriesFile}`, repositoriesArray, () => {
-      creator.createFile(`${config.tempLocation}/${config.notClonedRepositoriesFile}`, repositoriesFails, next);
+
```
Comment (mjasinski5) [inline src/integration/cloneDocuSources.js]: Just a suggestion to improve readability.
```
const repositoriesJson = JSON.stringify(repositoriesArray);
const repositoriesFailJson = JSON.stringify(repositoriesFails);

creator.createFile(`${config.tempLocation}/${config.indepenedentDocuRepositoriesFile}`, repositoriesJson, () => {

creator.createFile(`${config.tempLocation}/${config.notClonedRepositoriesFile}`, repositoriesFailJson, next);
      });		      });
```

**Pull request merged**